src/dev/repl_utils.clj

```diff
@@ -0,0 +1,19 @@
+(ns dev.repl-utils
+  (:require [time-tracker.core :as core]))
+
+;; Functions for easily starting, stopping and restarting the server in the REPL during development.
+
+(defonce server-stop-fn (atom nil))
+
+(defn repl-start-server! []
+  (let [stop-fn (core/start-server!)]
+    (swap! server-stop-fn (constantly stop-fn))))
```

Comment:
This can be `(reset! server-stop-fn stop-fn)`
Yeah I know, I should've written that myself